lsutils/__init__.py

```diff
@@ -0,0 +1,121 @@
+import git
+import github3
+import urllib3
+urllib3.disable_warnings()
+from githubutil import GitHubHelper
+import util
+from util import info, fail
+
+
+def create_submodule_update_pull_request(
+    repo_path: str,
+    repo_branch: str,
+    repo_owner_and_name: str,
+    submodule_path: str,
+    predecessor_repo_owner_and_name: str,
+    github_auth_token: str,
+    github_url: str='https://github.wdf.sap.corp',
+    github_verify_ssl: bool=False,
+):
+    '''
+    Create a pullrequest in the repository on the GitHub-instance which updates the submodule at the given path to
+    the commit that is current in the latest release of the predecessor repository.
+
+    Assumes that the submodule is at the same path in both repositories.
+    '''
+    # get information about submodule in the current repo
+    git_repo = git.Repo(repo_path)
+    submodule = find_submodule_by_path(git_repo, submodule_path)
+    current_submodule_sha = submodule.hexsha
+
+    p_username, p_repository = extract_owner_and_name(predecessor_repo_owner_and_name)
+    username, repository = extract_owner_and_name(repo_owner_and_name)
+
+    # get the relevant information about the submodule (i.e. the commit it's pointing to) from the predecessor
+    github = github3.github.GitHubEnterprise(
```

Comment:
pass-in github_api object. also, there is already a ready-to-use function that creates such an object in githubutil.py.